GoogleForCreators/web-stories-wp
# Add workspace icons

## Context

See #6204 for todo.

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Load up a story
2. Observe the new icons in the page navigation, page menu, carousel, and carousel menu
3. Verify that all the above-mentioned icons work as intended (as before)

### UAT




## Checklist




---

Fixes #6204
